kvm-install-vm

```diff
@@ -627,7 +627,7 @@ _EOF_
 
     outputn "Copying cloud image ($(basename ${IMAGE}))"
     DISK=${VMNAME}.qcow2
-    qemu-img create -f qcow2 -F qcow2 -b $IMAGE $DISK && ok
+    qemu-img create -q -f qcow2 -F qcow2 -b $IMAGE $DISK 10G && ok
```

Comment:
I think the `10G` got left behind here inadvertently.

Reply:
Sorry, I'll fix it.